privacy-snapshot.json

```diff
@@ -3,6 +3,7 @@
   "api.lens.dev",
   "api.segment.io",
   "api.web3modal.com",
+  "api.web3modal.org",
```

Comment:
why is this being added?

Reply:
It was added when running the e2e test locally.

Reply:
Can we double-check from which dependency the network request for this domain is coming from?

Reply:
Its coming from the snap account signature test
```
<testsuite name="Snap Account Signatures" timestamp="2024-05-30T07:18:47" tests="3" file="/Users/montelai/consensys/extension-2/test/e2e/accounts/snap-account-signatures.spec.ts" time="31.928" failures="3">
    <testcase name="Snap Account Signatures can sign with sync flow" time="31.290" classname="can sign with sync flow">
 <failure message="A new host not contained in the privacy-snapshot received a network
           request during test execution. Please update the privacy-snapshot
           file by passing the --update-privacy-snapshot option to the test
           command or add the new hosts to the snapshot manually.

           New hosts found: api.web3modal.org." type="Error"><![CDATA[Error: A new host not contained in the privacy-snapshot received a network
           request during test execution. Please update the privacy-snapshot
           file by passing the --update-privacy-snapshot option to the test
           command or add the new hosts to the snapshot manually.

           New hosts found: api.web3modal.org.
```

Reply:
This should be safe to revert. Not sure if we want to add it, we could

https://github.com/WalletConnect/web3modal/blob/712252de35446c10bb9413bf85ec5f2937dbd44c/packages/core/src/utils/CoreHelperUtil.ts#L188-L204

https://github.com/WalletConnect/web3modal/blob/712252de35446c10bb9413bf85ec5f2937dbd44c/packages/core/src/utils/ConstantsUtil.ts#L32